SourceX/dx.cpp

```diff
@@ -249,6 +250,77 @@ void LimitFrameRate()
 	frameDeadline = tc + v + refreshDelay;
 }
 
+#ifdef USE_SDL1
+
+void NewHWCursor(SDL_Surface *surf)
+{
+}
+
+#else
+
+static SDL_Surface *cursor_surface = NULL, *cursor_realsurface = NULL;
+static SDL_Cursor *cursor = NULL;
+static int cursor_width, cursor_height;
+static int cursor_surface_palette_version = -1;
+static int cursow_window_res_version = -1;

```

Comment:
Use consistent naming strategy. camelCase instead of `_`

Reply:
There's no consistency in regards to naming whatsoever. If camelCase is the agreed naming convention for devilutionX, that's fine but generally I try to follow the style of what's around the piece I'm changing so it doesn't stick out that much.